app/views/exchanges/hbx_profiles/people_index_datatable.html.erb

```diff
@@ -0,0 +1,10 @@
+<%= render "ui-components/v1/navs/primary_nav" %>
+<div class="container">
+  <h1 class="<%= @bs4 ? '' : 'heading-text'%>"><%= l10n('hbx_profiles.people.table.title') %></h1>
```

Comment:
@saipraveen18 We do not need `non-bs4` code anymore. We still have the `non-bs4` code in our codebase because the cucumbers depend on the `non-bs4` code. Please clean up this to not depend on bs4.